src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -100,7 +109,14 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
-            _inputObjects.Add(InputObject);
+            if (ExperimentalFeature.IsEnabled("Microsoft.PowerShell.Utility.PSConvertToJsonRaw") && Raw && InputObject is PSObject)
```

Comment:
```suggestion
            if (ExperimentalFeature.IsEnabled("Microsoft.PowerShell.Utility.PSConvertToJsonRaw")
                && Raw
                && InputObject is PSObject obj)
```

this along with using `obj` below.

Reply:
The call to `ExperimentalFeature.IsEnabled("Microsoft.PowerShell.Utility.PSConvertToJsonRaw")` is not needed here. `Raw` is guarded by the experimental attribute, so when the fetaure is disabled, a user cannot see `-Raw` and thus `Raw` will always be false.